docs/src/snap/reference/annotations.md

```diff
@@ -1,35 +1,179 @@
 # Annotations
 
 This page outlines the annotations that can be configured during cluster
-[bootstrap]. To do this, set the cluster-config/annotations parameter in
+[bootstrap]. To do this, set the `cluster-config/annotations` parameter in
```

Comment:
People typically expect dots to indicate nesting for json and yaml structures. So, `cluster-config.annotations`